.github/workflows/phpcs.yml

```diff
@@ -0,0 +1,32 @@
+name: phpcs
+
+on:
+  pull_request:
+    branches:
+      - trunk
+      - develop
+      - develop-v1
+
+jobs:
+  changed:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - id: changed-files
+        uses: tj-actions/changed-files@v19
+        with:
+          files: |
+            **/*.php
+      - name: Set PHP version
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.4
+          tools: composer:v2
+          coverage: none
+      - name: Install dependencies
+        if: ${{ steps.changed-files.outputs.any_changed == 'true' }}
```

Comment:
Is it possible to make this conditional on either files changed or a push to `trunk|develop|stable`?

I think it requires using the `github.event` object with an appropriate value from these docs https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows

Reply:
fixed: https://github.com/10up/distributor/pull/1075/commits/9ef796ad0bdc901568539abf5b5a433af3a883a7